docs/manual/java/releases/Migration14.md

```diff
@@ -52,6 +52,28 @@ cassandra-snapshot-store.keyspace = ${my-service.cassandra.keyspace}
 lagom.persistence.read-side.cassandra.keyspace = ${my-service.cassandra.keyspace}
 ```
 
+If you are using macOS or Linux, or have other access to a bash scripting environment, running this shell script in your base project directory can help add this configuration to existing services:
+
+```bash
+#!/bin/bash
+
+set -eu
+
+for svc_dir in *-impl; do
+    svc=${svc_dir%-impl}
+
+    # change $svc_dir to $svc on the next line to leave out the _impl suffix
+    keyspace=$(tr - _ <<<$svc_dir)
+    cat >> $svc_dir/src/main/resources/application.conf <<END
+$svc.cassandra.keyspace = $keyspace
```

Comment:
`$keyspace` may have characters not valid for a keyspace name (I think `-` is not valid but can't remember). I'd rather leave this blank and add an `echo "Review the value of $svc.cassandra.keyspace in file $svc_dir/src/main/resources/application.conf"`. Or something similar.

Actually, this is fine. If the character is invalid it won't start and user will have to come here and edit it. And if it works, well... it works.